[shelly] Improved Motion Support, Support CoIoT Unicast, fixes (#10220)
openhab/openhab-addons: bundles/org.openhab.binding.shelly/README.md

### Shelly Motion (thing-type: shellymotion)

Important: The Shelly Motion does only support CoIoT Unicast, which means you need to set the CoIoT peer address.

- Use device WebUI, open COIOT settings, make sure CoIoT is enabled and enter the openHAB IP address or

> - Use [Shelly Manager](doc/ShellyManager.md, select Action 'Set CoIoT peer' and the Manager will sets the openHAB IP address as peer address

Reviewer: ```suggestion
- Use [Shelly Manager](doc/ShellyManager.md), select Action 'Set CoIoT peer' and the Manager will sets the openHAB IP address as peer address
```

Author: remove, this belongs to the next PR

Reviewer: Did you push your change?

|          |illumination   |String   |yes      |Current illumination: dark/twilight/bright                           |
|          |vibration      |Switch   |yes      |ON: Vibration detected                                               |
|          |charger        |Switch   |yes      |ON: USB charging cable is connected external power supply activated. |

> |          |notionActive   |Switch   |yes      |ON: Motion detection is currently active                             |

Reviewer: ```suggestion
|          |motionActive   |Switch   |yes      |ON: Motion detection is currently active                             |
```

Author: fixed

|          |lastUpdate     |DateTime |yes      |Timestamp of the last update (any sensor value changed)              |
|battery   |batteryLevel   |Number   |yes      |Battery Level in %                                                   |
|          |lowBattery     |Switch   |yes      |Low battery alert (< 20%)                                            |
|device    |sensorSleepTime|Number   |no       |Specifies the number of sec the sensor should not report events      ]

Use case for the 'sensorSleepTime': 
You have a Motion controlling your light. 
You switch off the light and want to leave the room, but the motion sensor immediately switches light back on.

> Using 'sensorSleepTime' you could suspress motion events while leaving the room, e.g. for 5sec and the light doesn's switch on. 

Reviewer: ```suggestion
Using 'sensorSleepTime' you could suppress motion events while leaving the room, e.g. for 5sec and the light doesn't switch on. 
```

Author: fixed

Author: fixed
